app/views/course/courses/index.html.slim

```diff
@@ -1,5 +1,10 @@
 = page_header do
-  = new_button([:course]) if can?(:create, Course)
+  - if can?(:create, Course.new)
+    = new_button([:course])
+  - elsif request = current_tenant.user_role_requests.find_by(user_id: current_user&.id)
```

Comment:
Cannot hide the course creation button for instructor here, this request button will only be shown if the user cannot create course.

For instructor's request to be an admin, I am not sure if the user case is strong enough. Even we want to do that, the button will be somewhere else and the UI might be slightly different.